drivers/hmis/app/graphql/schema.graphql

```diff
@@ -222,11 +227,6 @@ type AutofillValue {
   Value to autofill if condition is met
   """
   valueNumber: Int
-
-  """
-  Link ID of question to autofill the value from if condition is med
-  """
-  valueQuestion: String
```

Comment:
This wasn't being used, I'm not sure what I added it for. I can't think of a case for autofilling the exact answer from another question, and it's not supported in the frontend anyway. So removing it for now.